docs/topics/game/game.md

```diff
@@ -0,0 +1,123 @@
+---
+description: Game
+---
+
+# Game
+
+## Introduction
+
+A [GAME]() is a non-fungible ERC-721 token that can be placed on a specific [LAND](../land/land.md) shape. The game will be created in the GameMaker and the creator is able to add [ASSETs](../asset/asset.md) to it to populate its game world.
+
+Learn more about GAME in The Sandbox Metaverse in this [article]().
+
+### Class diagram
+
+```plantuml
+
+class ERC721BaseToken {
+    {field} + _numNFTPerAddress: mapping (address => uint256)
+    {field} + _owners: mapping (uint256 => uint256)
+    {field} + _operatorsForAll: mapping (address => mapping(address => bool))
+    {field} + _operators: mapping (uint256 => address)
+
+    + balanceOf(owner)
+    + ownerOf(id)
+    + approveFor(sender, operator, id)
+    + approve(operator, id)
+    + getApproved(id)
+    + transferFrom(from, to, id)
+    + safeTransferFrom(from, to, id)
+    + batchTransferFrom(from, to, ids, data)
+    + safeBatchTransferFrom(from, to, ids, data)
+    + supportsInterface(id)
+    + setApprovalForAllFor(sender,operator,approved)
+    + setApprovalForAll(operator, approved)
+    + isApprovedForAll(owner, operator)
+    + burn(id)
+    + burnFrom(from, id)
+
+
+}
+
+class ImmutableERC721 {
+    {field} + CREATOR_OFFSET_MULTIPLIER: uint256
+    {field} + SUBID_MULTIPLIER: uint256
+    {field} + CHAIN_INDEX_OFFSET_MULTIPLIER: uint256
+    {field} + STORAGE_ID_MASK: uint256
+    {field} + VERSION_MASK: uint256
+    {field} + CHAIN_INDEX_MASK: uint256
+    {field} + base32Alphabet: uint256
+    {field} + _chainIndex: uint8
+}
+
+class SuperOperator{
+    {field} # _superOperators: mapping (address => bool)
+    + setSuperOperator(superOperator, enabled)
+    + isSuperOperator(who)
+}
+
+class MetaTransactionReceiver{
+    {field} # _metaTransactionContracts: mapping (address => bool)
+    + isMetaTransactionProcessor(who)
+}
+
+interface IGameToken{
+    + createGame(from,to,creation,editor,subId)
+    + burn(gameId)
+    + burnFrom(from,gameId)
+    + recoverAssets(from,to,gameId,assetIds)
+    + burnAndRecover(from,to,gameId,assetIds)
+    + updateGame(from,gameId,update)
+    + getAssetBalances(gameId,assetIds)
+    + setGameEditor(gameCreator,editor,isEditor)
+    + isGameEditor(gameOwner,editor)
+    + creatorOf(id)
+    + transferCreatorship(sender,original,to)
+    + name()
+    + symbol()
+    + tokenURI()
+    + onERC1155Received(operator,from,id,value,data)
+    + onERC1155BatchReceived(operator,from,ids,values,data)
+}
+
+class GameBaseToken{
```

Comment:
Another class should inherit  `GameBaseToken`
`GameV1` ? (side question: where do we put the version of the implementation ?)

Reply:
What do you mean by saying version of implementation ? it's part of the proxy pattern I guess.

Reply:
There was a naming convention by having the version of the implementation in the class name, ie
`AssetV2`
So i'm suggesting to reuse the same approach here.

Reply:
So, as this is the first version of Game we'll name it GameV1?

Reply:
Added a GameV1 contract